src/locales/en/translations.json

```diff
@@ -0,0 +1,8 @@
+{
+  "en": {
+    "pagination": {
+      "first": "First",
+      "last":"Last"
+    }
+  }
+}
```

Comment:
Would we end up creating a json file in neptune or can we reuse the existing yml files?

Reply:
This file is where we define the format for this package, so In Neptune app: 
1. we need to re-define follow this file in Neptune to generate another language file: https://github.com/rdytech/neptune/pull/8727/files#diff-a5a86160b667645873b275b84f4524f4ade7347bbbd25813378cb148d0826be3R1333-R1335
2. `docker-compose run --rm app rake i18n:js:export` to generate translation for frontend side